its/ruling/src/test/java/org/sonar/python/it/PythonPrAnalysisTest.java

```diff
@@ -177,7 +179,8 @@ private SonarScanner prepareScanner(File path, String projectKey, String scenari
       .setProperty("sonar.cpd.exclusions", "**/*")
       .setProperty("sonar.lits.differences", litsDifferencesFile.getAbsolutePath())
       .setProperty("sonar.internal.analysis.failFast", "true")
-      .setEnvironmentVariable("SONAR_RUNNER_OPTS", "-Xmx2000m");
+      .setDebugLogs(true)
+      .setEnvironmentVariable("SONAR_SCANNER_OPTS", "-Xdebug");
```

Comment:
_minor_ Do we need both?

Reply:
No we don't, and in fact I should've kept the `.setEnvironmentVariable("SONAR_RUNNER_OPTS", "-Xmx2000m");` option. I mixed up the two when trying various ways to enable debug logs.